src/sdk/bucketspace_s3.js

```diff
@@ -43,7 +44,7 @@ class BucketSpaceS3 {
         try {
             const { name } = params;
             console.log(`bss3: read_bucket ${name}`);
-            await this.s3.headBucket({ Bucket: name }).promise();
+            await this.s3.headBucket({ Bucket: name });
```

Comment:
@guymguym I noticed that we don't do something with the `headBucket` result (you add a suggestion or change it in a future PR when you use `BucketSpaceS3`).